lib/views/home/home_view.dart

```diff
@@ -113,28 +114,29 @@ class _HomeViewState extends State<HomeView> {
   }
 
   Future<List<dynamic>?> listPayments() async {
-    final invoicesList = await widget.provider.get<AppApi>().listInvoices();
     final fundsList = await widget.provider.get<AppApi>().listFunds();
+    final listPays = await widget.provider.get<AppApi>().listTransaction();
```

Comment:
why we need list transaction? you already had the transaction in the listfunds right?

Reply:
Yes, it is included in the listfunds. I guess it would be much clear to deal with the funds and transactions seperately. What should I do should I converge both the outputs from the same json?